components/eamxx/mkdocs.yml

```diff
@@ -13,6 +13,7 @@ nav:
     - 'Regionally Refined EAMxx': 'user/rrm_eamxx.md'
     - 'Doubly Periodic EAMxx': 'user/dp_eamxx.md'
     - 'PySCREAM': 'user/pyscream.md'
+    - 'IO Metadata': 'user/io_metadata.md'
```

Comment:
I'm not seeing this rendered in the PR version of the docs.

Reply:
Its not on this page:  https://docs.e3sm.org/E3SM/pr-preview/pr-6957/EAMxx/user/